doc/model_persistence.rst

```diff
@@ -4,6 +4,47 @@
 Model persistence
 =================
 
+.. list-table:: Summary of model persistence methods
+   :widths: 25 50 50
+   :header-rows: 1
+
+   * - Persistence method
+     - Pros
+     - Risks / Cons
+   * - :ref:`ONNX <onnx_persistence>`
+     - * Serve models without a Python environment
+       * Serving and training environments independent of one another
+       * Most secure option
+     - * Not all scikit-learn models are supported
+       * Custom estimators require more work to support
+       * Original Python object is lost and cannot be reconstructed
+   * - :ref:`skops_persistence`
+     - * More secure than `pickle` based formats
+       * Contents can be partly validated without loading
+     - * Not as fast as `pickle` based formats
+       * Supports less types than `pickle` based formats
+       * Requires the same environment as the training environment
+   * - :mod:`pickle`
+     - * Native to Python
+       * Can serialize most Python objects
+       * Efficient memory usage with `protocol=5`
+     - * Loading can execute arbitrary code
+       * Requires the same environment as the training environment
+   * - :mod:`joblib`
+     - * Efficient memory usage
+     - * Supports memory mapping
```

Comment:
There are now 4 columns for this row because of my previously careless suggestion. Hopefully this should fix the broken table rendering.


```suggestion
       * Supports memory mapping
```